cpp/cmake/EvalGpuArchs.cmake

```diff
@@ -58,6 +60,6 @@ int main(int argc, char** argv) {
     OUTPUT_VARIABLE __gpu_archs
     OUTPUT_STRIP_TRAILING_WHITESPACE
     ERROR_FILE ${error_file})
-  message("Auto detection of gpu-archs: ${__gpu_archs}")
+    message(VERBOSE "CUDF: Auto detection of gpu-archs: ${__gpu_archs}")
```

Comment:
I think the spacing on this was right as it happened at the same scope as the `execute_process` call.

Reply:
Correct, revised.

Reply:
The current revision fails when the CMake minimum required is 3.18 or greater. 

Going to need some more logic to the `project` preamble.

Reply:
Updated, and now this works without change when CUDF cmake minimum required moves to 3.18.